plutus-playground-client/src/MainFrame.purs

```diff
@@ -464,10 +463,7 @@ handleActionForm initialValue = rec
 
   rec (SetSubField n subEvent) s@(FormObject fields) = FormObject $ over (ix n <<< _Newtype <<< _2) (rec subEvent) fields
 
-  rec AddSubField (FormArray schema fields) =  -- As the code stands, this is the only guarantee we get that every -- value in the array will conform to the schema: the fact that we -- create the 'empty' version from the same schema template. --
-    -- Is more type safety than that possible? Probably.
-    -- Is it worth the research effort? Perhaps. :thinking_face:
-    FormArray schema $ Array.snoc fields (toArgument initialValue schema)
+  rec AddSubField (FormArray schema fields) =  -- As the code stands, this is the only guarantee we get that every -- value in the array will conform to the schema: the fact that we -- create the 'empty' version from the same schema template. -- -- Is more type safety than that possible? Probably. -- Is it worth the research effort? Perhaps. :thinking_face: FormArray schema $ Array.snoc fields (toArgument initialValue schema)
```

Comment:
This is the only thing that bothers me. Can we rearrange this comment somehow so Purty doesn't clobber it?